raft/src/test/java/org/apache/kafka/raft/FileQuorumStateStoreTest.java

```diff
@@ -64,7 +66,7 @@ void testWriteReadElectedLeader(short kraftVersion) throws IOException {
         );
 
         final Optional<ElectionState> expected;
-        if (kraftVersion == 1) {
+        if (kraftVersion.featureLevel() > 0) {
```

Comment:
If you agree with one of my previous message about adding `KRaftVersion::isReconfigSupported` maybe we can use that predicate here. This comment applies to a few lines in `src/test`.